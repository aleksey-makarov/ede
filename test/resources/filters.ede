{
  "text": "Lorem Ipsum",
  "bool": true,
  "number": 1.0
}
---
{{ text | lower }}
{{ text | lower | upper }}
{{ bool | show | lower }}
{{ bool | show | upper }}
{{ number | show | lower | upper }}

{% if text | lower == "lorem ipsum" %}
text | lower == "lorem ipsum"
{% endif %}
