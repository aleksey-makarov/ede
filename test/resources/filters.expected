lorem ipsum
LOREM IPSUM
true
TRUE
1.0

text | lower == "lorem ipsum"
